### ISSUE TITLE
Bump value of __cpp_lib_allocate_at_least for C++23

### ISSUE THREAD

**Issue by jwakely** [cplusplus/draft#6202]
[LWG 3887](https://cplusplus.github.io/LWG/issue3887) wants to increase the value of the `__cpp_lib_allocate_at_least` macro to distinguish the pre-P2652R0 API from the new one that will be in C++23. LWG voted this Tentatively Ready, but if we follow the usual process that will be approved in Varna, for C++26. There's not much point updating it if we only do so for C++26.

I think it can be argued that the value of the macro in the standard is editorial. The standard doesn't say what a given value means, only [SD-6](https://isocpp.org/std/standing-documents/sd-6-sg10-feature-test-recommendations#__cpp_lib_allocate_at_least) does that.

Would the editor please consider giving the value `202302L` to `__cpp_lib_allocate_at_least`, and we'll get @brevzin to update SD-6?

**Comment by jwakely** [comment]
@phalpern - as the author of [P2652](https://wg21.link/p2652) please indicate your support (or not) for this.

**Comment by brevzin** [comment]
Yep, this is a good idea 👍 . We've also done this with other papers in the past too.

**Comment by tkoeppe** [comment]
That seems reasonable! Could you please send a PR? We will land that in the next draft, together with the review committee feedback.

**Comment by phalpern** [comment]
Yes, I agree
